src/services/prisma-datastore/db-init.ts

```diff
@@ -16,7 +16,6 @@ import {
 
 const prisma = new PrismaClient();
```

Comment:
To trigger this db change in production, we should change our netlify.toml. Thusly add a section like this

```toml
[context.production]
  command = "sed -i \"s|process.env.DEPLOY_URL|'${DEPLOY_URL}'|g\" config.ts && yarn build && yarn db:update"
```

that is, I added the `&& yarn db:update` to the default build command.